src/containers/ArticlePage/FrontpageArticlePage/components/FrontpageArticlePanels.tsx

```diff
@@ -37,6 +37,7 @@ const FrontpageArticlePanels = ({ article, handleSubmit, articleLanguage }: Prop
         title={t('form.contentSection')}
         className={'u-4/6@desktop u-push-1/6@desktop'}
         hasError={!!(errors.title || errors.introduction || errors.content)}
+        data-wide={true}
```

Comment:
Burde denne settes èn plass ytterst og så arves derfra?

Reply:
Kan legge det til om man skal ha en sjekk boks. For nå tror jeg det er greit å ha det på dette nivået. Ser ikke noen grunn til å propegate den nedover per nå